src/detect-engine-siggroup.c

```diff
@@ -630,7 +630,11 @@ void SigGroupHeadSetFilestoreCount(DetectEngineCtx *de_ctx, SigGroupHead *sgh)
             continue;
 
         if (SignatureIsFilestoring(s)) {
-            sgh->filestore_cnt++;
+            if (sgh->filestore_cnt == UINT16_MAX) {
+                SCLogError(SC_ERR_NOT_SUPPORTED, "Too many filestore in signature group head");
```

Comment:
@catenacyber think this was overlooked

Reply:
> like this it behaves like a warning. So it should either be an error we handle somehow, or a warning.

Warning in https://github.com/OISF/suricata/pull/9997

Not doing an error, because `SigAddressPrepareStage4` returning error is fatal
And just removing a sgh looks bad as well...

> think this was overlooked

Do you mean this comment ?
It was still in my pipeline ;-)